R/round.r

```diff
@@ -50,17 +52,33 @@
 #'   to be rounded to. Valid base units are `second`, `minute`, `hour`, `day`,
 #'   `week`, `month`, `bimonth`, `quarter`, `season`, `halfyear` and
 #'   `year`. Arbitrary unique English abbreviations as in the [period()]
-#'   constructor are allowed. Rounding to multiple of units (except weeks) is
+#'   constructor are allowed. Rounding to multiples of units (except weeks) is
 #'   supported.
-#' @param change_on_boundary If NULL (the default) don't change instants on the
-#'   boundary (`ceiling_date(ymd_hms('2000-01-01 00:00:00'))` is `2000-01-01
-#'   00:00:00`), but round up `Date` objects to the next boundary
-#'   (`ceiling_date(ymd("2000-01-01"), "month")` is `"2000-02-01"`). When
-#'   `TRUE`, instants on the boundary are rounded up to the next boundary. When
-#'   `FALSE`, date-time on the boundary are never rounded up (this was the
-#'   default for \pkg{lubridate} prior to `v1.6.0`. See section `Rounding Up
-#'   Date Objects` below for more details.
-#' @param week_start when unit is `weeks` specify the reference day; 7 being Sunday.
+#' @param change_on_boundary if this is `NULL` (the default), instants on the boundary
+#'   remain unchanged, but `Date` objects are rounded up to the next boundary.
+#'   If this is `TRUE`, instants on the boundary are rounded up to the next boundary.
+#'   If this is `FALSE`, nothing on the boundary is rounded up at all. This was the
+#'   default for \pkg{lubridate} prior to `v1.6.0`.
+#'   See section `Rounding Up Date Objects` below for more details.
+#'
+#'   For example, as per default behaviour `NULL`,
```

Comment:
I like the way the behavior is detailed here. Could you please convert to examples and add code just before the `@export` ?